### PR TITLE
chore(tooltips): update prop descriptions

### PR DESCRIPTION
**Pull request by Mikaelia**
## Description
Updates tooltips package prop descriptions

### CODE REVIEW

File: packages/tooltips/src/elements/Tooltip.tsx [hunk 1]
```diff
@@ -25,30 +25,37 @@ import { StyledTooltipWrapper, StyledTooltip, TOOLTIP_SIZE, TOOLTIP_TYPE } from
...
+  /** Allows the tooltip to reposition during browser resize events */
   eventsEnabled?: boolean;
+  /** Sets the ID of the tooltip */
   id?: string;
+  /** Defines the content of the tooltip */
   content: React.ReactNode;
   /**
-   * These placements differ from the default naming of Popper placements to help
+   * Adjusts the placement of the tooltip. Placement values differ from Popper placement values to
    * assist with RTL layouts.
```
Comment (jzempel) [inline packages/tooltips/src/elements/Tooltip.tsx:42]: Again, I think the simplicity of your previous description is ideal:

https://github.com/zendeskgarden/react-components/pull/957/files#diff-8fc5bdeb0d77c73a3b03ce92de8df5ff9a393a3b092cc638a7700df6c8f42ec3L30-L33

File: packages/tooltips/src/elements/Tooltip.tsx [hunk 2]
```diff
@@ -25,30 +25,37 @@ import { StyledTooltipWrapper, StyledTooltip, TOOLTIP_SIZE, TOOLTIP_TYPE } from
...
    **/
   placement?: GARDEN_PLACEMENT;
-  /** See Popper [documentation](https://popper.js.org/docs/v1/#modifiers--codeobjectcode) for details */
+  /** Passes configurations to the [Popper instance](https://popper.js.org/docs/v1/#modifiers--codeobjectcode) */
```
Comment (jzempel) [inline packages/tooltips/src/elements/Tooltip.tsx:45]: Let's stick with the updated `https://popper.js.org/docs/v2/modifiers/` URL.

File: packages/tooltips/src/elements/Tooltip.tsx [hunk 3]
```diff
@@ -25,30 +25,37 @@ import { StyledTooltipWrapper, StyledTooltip, TOOLTIP_SIZE, TOOLTIP_TYPE } from
...
   popperModifiers?: Modifiers;
+  /** Adjusts the padding and font size */
   size?: TOOLTIP_SIZE;
+  /** Applies either light or dark styling */
```
Comment (jzempel) [inline packages/tooltips/src/elements/Tooltip.tsx:49]: This feels a wee bit redundant based on what we see here: https://garden.zendesk.com/components/tooltip#tooltip

File: packages/tooltips/src/elements/Tooltip.tsx [hunk 4]
```diff
@@ -25,30 +25,37 @@ import { StyledTooltipWrapper, StyledTooltip, TOOLTIP_SIZE, TOOLTIP_TYPE } from
...
   type?: TOOLTIP_TYPE;
+  /** Sets the `z-index` of the tooltip */
   zIndex?: number | string;
-  /** Determine visibility on initial render */
+  /** Displays the tooltip on initial render */
   isInitialVisible?: boolean;
-  /** Control visibility state of the Tooltip */
+  /** Displays the tooltip */
   isVisible?: boolean;
   children: React.ReactElement;
```
Comment (jzempel) [inline packages/tooltips/src/elements/Tooltip.tsx:57]: Good call not documenting. Can this be removed from the interface without prompting a TS error? Generally, we don't define default props.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: packages/tooltips/src/elements/Tooltip.tsx (hunk 4 above)
Comment (Mikaelia) [reply packages/tooltips/src/elements/Tooltip.tsx:56]: Looks like it breaks a test

File: packages/tooltips/src/elements/Tooltip.tsx (hunk 3 above)
Comment (Mikaelia) [reply packages/tooltips/src/elements/Tooltip.tsx:48]: Would something like "Specifies the tooltip type" be any better?

Comment (jzempel) [reply packages/tooltips/src/elements/Tooltip.tsx:48]: Yeah, I think so.

**Review by jzempel — APPROVED** [review]

**Pull request merged**